spec/gist_updater/debug_spec.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+describe GistUpdater do
+  describe '.debug' do
+    it { is_expected.to respond_to :debug }
+    it { is_expected.to respond_to :debug= }
+  end
```

Comment:
See http://stackoverflow.com/questions/24434381/ruby-rspec-how-should-i-be-testing-attr-accessor